nuopc_dart.F90

```diff
@@ -24,7 +24,7 @@ module dart_comp_nuopc
     use NUOPC            , only : NUOPC_CompAttributeGet, NUOPC_Advertise
     use NUOPC            , only : NUOPC_CompFilterPhaseMap
     use NUOPC, only : NUOPC_Write ! HK what is the write for?
```

Comment:
`NUOPC_Write` is to write the fields which is assigned as export state in the netcdf file. So this would be the assimilated data from DART.